frontend/src/components/Table/Table.tsx

```diff
@@ -274,7 +275,7 @@ const Table = <T extends IIdentifiedObject, TFilter extends object = {}>(
         ? { sortBy, pageIndex: pageIndexProp ?? 0, pageSize: pageSizeProp }
         : { sortBy, pageIndex: pageIndexProp ?? 0 },
       manualPagination: manualPagination ?? true, // Tell the usePagination hook
-      manualSortBy: false,
+      manualSortBy: manualSortBy,
```

Comment:
This fixes a bug where the FE was attempting to perform a sort on data already sorted by the backend.